src/Sylius/Bundle/ShopBundle/Provider/PaymentRequestPayResponseProvider.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Provider;
+
+use Sylius\Bundle\ResourceBundle\Controller\RequestConfiguration;
+use Sylius\Component\Core\Model\OrderInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+final class PaymentRequestPayResponseProvider implements PayResponseProviderInterface
+{
+    public function getResponse(
+        RequestConfiguration $requestConfiguration,
+        OrderInterface $order
+    ): Response {
+        return new Response('', Response::HTTP_NO_CONTENT);
```

Comment:
I've seen this more modern sytax before, but not sure if it will work.

```suggestion
        return new Response(status: Response::HTTP_NO_CONTENT);
```

Reply:
This line are just here to fill out the blanks right now. I will think about using the new named arguments when severals args have default values, it seems more cleaner indeed. Thank you for pointing this out @diimpp